config/routes.rb

```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  
-  devise_for :users
-  # Defines the root path route ("/")
-  root "users#index"
+
+  # Web routes
+
+  devise_for :users # Routes for authentication
+ 
+  root "users#index"  # Defines the root path route ("/")
 
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :new, :create, :show, :destroy] do
```

Comment:
### Optional
- [ ] We currently have duplicated routes which is making the routes look messy and difficult to understand